ffi/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ endif()
 add_library(llvmlite SHARED assembly.cpp bitcode.cpp core.cpp initfini.cpp
             module.cpp value.cpp executionengine.cpp transforms.cpp type.cpp
             passmanagers.cpp targets.cpp dylib.cpp linker.cpp object_file.cpp
-            custom_passes.cpp orcjit.cpp memorymanager.cpp)
+            custom_passes.cpp orcjit.cpp memorymanager.cpp newpassmanagers.cpp)
```

Comment:
We'll soon (relatively) be in a position where there are only the new pass managers, so I think it would be better to rename the existing `passmanagers.cpp` to `legacypassmanagers.cpp`, and make the new file `passmanagers.cpp`.

Reply:
done in latest revision

Reply:
(note that we undid this later - resolving the conversation)